client/components/popover/util.js

```diff
@@ -333,17 +336,19 @@ function _offset( box, doc ) {
 	};
 }
 
-/*
+/**
  * Constrain a left to keep the element in the window
- * @param  {Object} pl proposed left
- * @param  {Number} ew tip element width
- * @return {Number}    the best width
+ *
+ * @param {object} off Proposed offset before constraining
+ * @param {window.Element} el Element to be constained to viewport
+ * @returns {number}    the best width
  */
-const constrainLeft = function( off, el ) {
+function constrainLeft( off, el ) {
+	const viewport = getViewport();
 	const ew = getBoundingClientRect( el ).width;
 	off.left = Math.max( 0, Math.min( off.left, viewport.width - ew ) );
 
 	return off;
-};
+}
 
 export { constrainLeft, bindWindowListeners, unbindWindowListeners, suggested, offset };
```

Comment:
I know this code was already there, but consider exporting each function directly. It's much easier to figure out what's private and public that way.

Reply:
OK, I can change that.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/38528/commits/9197112f07de0004784f2aee7e4b8fa399584162

Reply:
Thank you! 👍